docs/public-networks/reference/cli/options.md

```diff
@@ -2584,7 +2583,7 @@ UPnP support is often disabled by default in networking firmware. If disabled by
 
 :::tip
 
-Use `UPNPP2PONLY` if you wish to enable UPnP for p2p traffic but not JSON-RPC.
+Use `UPNPP2PONLY` if you wish to enable UPnP for P2P traffic but not JSON-RPC.
```

Comment:
I would take "wish" out

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/want

Maybe something like: 

```suggestion
Use `UPNPP2PONLY` to enable UPnP for P2P traffic but not JSON-RPC.
```

Reply:
Er -- don't want to stray too far from issue >> this will need its own issue. My OC just couldn't help itself and corrected p2p